docs/spec/operators/socketenricher.md

```diff
@@ -0,0 +1,21 @@
+---
+title: SocketEnricher
+---
+
+The Socket enricher operator provides information about the socket owner, i.e.
+the process that created the socket. This is used to correlate sockets and
+process information in gadgets that only have information about the socket, like
+networking ones. The following information can be get from a socket object:
```

Comment:
```suggestion
networking ones. The following information can be gotten from a socket object:
```

Reply:
I know that's the right form but it sounds very weird to me.